lib/tasks/data_maintenance.rake

```diff
@@ -24,7 +24,7 @@ namespace :data_maintenance do
     Location.prisons.each do |prison|
       from_moves = prison.moves_from.requested.select(&:from_nomis?).each do |move|
         move.update!(status: Move::MOVE_STATUS_CANCELLED,
-                     cancellation_reason: Move::MOVE_CANCELLATION_REASON_MADE_IN_ERROR)
+                     cancellation_reason: Move::cancellation_reasons[:made_in_error])
```

Comment:
I think we might need to use `CANCELLATION_REASON_MADE_IN_ERROR` here instead as we are no longer using the hash?

Reply:
oops, good spot 👍